packages/design-system/src/components/Stack/StackItem.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { forwardRef, ReactNode } from 'react';
+import classnames from 'classnames';
+import styles from './StackItem.module.scss';
+
+export const alignOptions = {
+	auto: 'align-auto',
+	start: 'align-start',
+	end: 'align-end',
+	center: 'align-center',
+	stretch: 'align-stretch',
+	baseline: 'align-baseline',
+};
+
+export const overflowOptions = {
+	scroll: 'overflow-scroll',
+	hidden: 'overflow-hidden',
+	visible: 'overflow-visible',
+	auto: 'overflow-auto',
+};
+
+export const possibleAsTypes = ['div', 'li', 'span', 'section', 'dd', 'dt'] as const;
+
+export type ItemProps = {
+	children: ReactNode;
+	grow?: boolean;
+	shrink?: boolean;
+	align?: keyof typeof alignOptions;
+	overflow?: keyof typeof overflowOptions;
+	as?: typeof possibleAsTypes[number];
+};
+
+export const StackItem = forwardRef(function StackItem(
+	{
+		as = 'div',
+		children,
+		grow = false,
+		shrink = true,
```

Comment:
Why is it true ?
In the css, the default item shrink is 0. But in the code you force it a 1 except if the user force it at 0.

Reply:
Initial flex-shrink is 1 https://developer.mozilla.org/en-US/docs/Web/CSS/flex-shrink

Reply:
ok 👍